.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": ["airbnb-base", "plugin:prettier/recommended"],
+  "plugins": ["import"],
+  "rules": {
+    "quotes": ["error", "double", {"avoidEscape": true}],
+    "vars-on-top": "off",
```

Comment:
This is not needed anymore

Reply:
What do you mean? My editor was complaining about the rule, which I find ridiculous, so I disabled it.